components/Profile/index.jsx

```diff
@@ -46,21 +75,17 @@ const ProfileBody = ({ profile }) => {
       <Row gutter={48}>
         <Col className="mb-48">
           <Title level={4}>Badge</Title>
-          <BadgeCard>
-            {details?.image ? (
-              <Image
-                src={getAutonolasTokenUri(details.image)}
-                alt="Badge image"
-                width={IMAGE_SIZE}
-                height={IMAGE_SIZE}
-                className="nft-image"
-                preview={false}
-              />
-            ) : (
-              <Skeleton active />
-            )}
-          </BadgeCard>
-          {details && !details.image && <Text>Badge not minted yet</Text>}
+          {isBadgeLoading ? (
+            <BadgeLoading />
+          ) : (
+            <>
+              {details?.image ? (
+                <ShowBadge image={details?.image} tokenId={details?.tokenId} />
+              ) : (
+                <Text>Badge not minted yet</Text>
```

Comment:
Let's point people to the leaderboard here so they can mint:

You don't have a badge – mint one

With mint one as a link to /leaderboard